lib/csv.rb

```diff
@@ -959,6 +959,12 @@ def initialize(encoding, line_number)
     write_empty_value:  "",
   }.freeze
 
+  module NoQuotedFields # :nodoc:
+    def self.[](_index)
+      false
+    end
+  end
```

Comment:
Could you move this to `lib/csv/fields_converter.rb` and define as a singleton `Array` object?

```ruby
NO_QUOTED_FIELDS = []
def NO_QUOTED_FIELDS.[](_index)
  false
end
NO_QUOTED_FIELDS.freeze
```

Reply:
Done!